tests/rt.conf

```diff
@@ -298,10 +298,10 @@ RUN | datm_cdeps_lnd_era5                               | - wcoss2           | b
 RUN | datm_cdeps_lnd_era5_rst                           | - wcoss2 noaacloud |          | datm_cdeps_lnd_era5
 
 ### ATM-LND tests ###
-COMPILE | atml | intel | -DAPP=ATML -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v17_p8,FV3_GFS_v17_p8_rrtmgp,FV3_GFS_v15_thompson_mynn_lam3km,FV3_WoFS_v0,FV3_GFS_v17_p8_mynn -D32BIT=ON | | fv3 |
-RUN | control_p8_atmlnd_sbs                             | - noaacloud | baseline |
-RUN | control_p8_atmlnd                                 | - noaacloud | baseline |
-RUN | control_restart_p8_atmlnd                         | - noaacloud |          | control_p8_atmlnd
+##COMPILE | atml | intel | -DAPP=ATML -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v17_p8,FV3_GFS_v17_p8_rrtmgp,FV3_GFS_v15_thompson_mynn_lam3km,FV3_WoFS_v0,FV3_GFS_v17_p8_mynn -D32BIT=ON | | fv3 |
+##RUN | control_p8_atmlnd_sbs                             | - noaacloud | baseline |
+##RUN | control_p8_atmlnd                                 | - noaacloud | baseline |
+##RUN | control_restart_p8_atmlnd                         | - noaacloud |          | control_p8_atmlnd
```

Comment:
@uturuncoglu @barlage I believe the current ATML regression tests will need to be updated. The current tests we have are being turned off in this PR. @dpsarmie what is the issue with these ATML tests?

Reply:
@uturuncoglu @DeniseWorthen :
There was a conflict with the pathing for the surface parameters and the new export_tiled feature. The default doesn't have the './INPUT/' subdirectory when building the path to the files. I went back and changed the tests to just overwrite this since it is a minor conflict. The latest version of theis branch should now have those **atmlnd** runs running successfully without any needs to update the baseline.
> PASS -- COMPILE 'atml_intel' [13:08, 11:27]
> PASS -- TEST 'control_p8_atmlnd_sbs_intel' [06:55, 04:14](1609 MB)
> PASS -- TEST 'control_p8_atmlnd_intel' [06:54, 04:11](1611 MB)
> PASS -- TEST 'control_restart_p8_atmlnd_intel' [04:37, 02:14](903 MB)

Reply:
@dpsarmie Thanks. We actually have a pending bug-fix PR for these tests #2191 and we could make the required change then. @uturuncoglu Is there any reason the tileX files need to be in INPUT vs the run directory?

Reply:
@DeniseWorthen @dpsarmie No. We could change them. In this case I just use standalone atm test as base. The fixed files required for the land component is just for C96 ones and there is no need to define them in input.nml. The cap finds them based on the input directory specific in the land section of the ufs.configure. I think input.nml needs to be consistent with the control_p8